blockchain/block/blockstore.go

```diff
@@ -41,9 +40,8 @@ func (in *Store) ToProto() *iotextypes.BlockStore {
 }
 
 // FromProto converts from proto message
-func (in *Store) FromProto(pb *iotextypes.BlockStore) error {
-	// TODO: pass the correct EVM network ID at the time of newFileDAOv2()
-	blk, err := (&Deserializer{}).SetEvmNetworkID(config.EVMNetworkID()).FromBlockProto(pb.Block)
+func (in *Store) FromProto(evmNetworkID uint32, pb *iotextypes.BlockStore) error {
+	blk, err := (&Deserializer{}).SetEvmNetworkID(evmNetworkID).FromBlockProto(pb.Block)
```

Comment:
moreover, I don't see the value of creating a new `Deserializer` in each function call